hw03_sethutsa.py

```diff
@@ -0,0 +1,197 @@
+#################################################################################
+# Author: Utsa Seth
+# Username: sethutsa
+#
+# Assignment: HW 03
+# Purpose: Make a turtle artwork based on guidelines
+# Google Doc Link: https://docs.google.com/document/d/1eaKOWOABEuIIrmJ1DmS89uaWYTad68SuiMbo4g9Pe5M/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: Learned how to add images and text from t03_functions_house.py
+# provided with t03 assignment. Took the idea of changing Dory's "just keep swimming"
+# to "just keep coding" from Nicholas Hamilton.
+#
+#################################################################################
+import turtle
+
+
+def draw_fish(t):
+    t.pensize(4)
+    t.color('yellow')
+    t.lt(90)
+    t.begin_fill()
+    for i in range(3):
+        t.forward(60)
+        t.right(120)
+    t.end_fill()
+    t.lt(75)
+    t.color('blue')
+    t.begin_fill()
+    for i in range(2):
+        t.fd(120)
+        t.rt(150)
+    t.end_fill()
+    t.color('yellow')
+    t.begin_fill()
+    t.right(30)
+    t.fd(150)
+    for i in range(2):
+        t.rt(120)
+        t.fd(30)
+    t.end_fill()
+    """
+    draws orange and pink fish using ocean
+    """
+
+def draw_fish_2(t):
```

Comment:
Can you reuse draw_fish to draw both fish, instead of a function for every fish (same for below with bubbles and starfish).